dctest-meows-runner/Dockerfile

```diff
@@ -48,14 +50,22 @@ RUN apt-get update \
             iptables \
             openssh-client \
             sudo \
+            hub \
     && rm -rf /var/lib/apt/lists/* \
     && curl -sSLf https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | tar -C /usr/local -xzf - \
     && curl -sfL https://github.com/cybozu-go/placemat/releases/download/v${PLACEMAT_VERSION}/placemat2_${PLACEMAT_VERSION}_amd64.deb -o placemat2_${PLACEMAT_VERSION}_amd64.deb \
     && dpkg -i ./placemat2_${PLACEMAT_VERSION}_amd64.deb \
     && rm ./placemat2_${PLACEMAT_VERSION}_amd64.deb \
+    && git clone -b ${NECO_BRANCH:-release} https://github.com/cybozu-go/neco.git ${NECO_DIR} \
+    && make -C ${NECO_DIR}/dctest setup \
+    && git clone -b ${NECO_APPS_BRANCH:-release} https://github.com/cybozu-go/neco-apps.git ${NECO_APPS_DIR} \
+    && make -C ${NECO_APPS_DIR}/test setup \
```

Comment:
Why do you clone here?

`neco` and `neco-apps` will be updated frequency.
Also, in the near future, they will be moved to the `cybozu-private` org and be integrated with `neco-apps-secret`.
This cloning makes risk to upload secrets to quay.io.

So I think they should be cloned at runtime.